### PR TITLE
Update errors.md to include examples of when ssl errors are thrown

### PR DESCRIPTION
**Pull request by thang62590**
This commit adds examples for when particular SSL errors are thrown.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/synthetics/api_tests/errors.md [hunk 1]
```diff
@@ -48,6 +48,6 @@ SSL errors can occur during an API test run. They are different from failing ass
...
-| `UNABLE_TO_GET_ISSUER_CERT`          | Unable to find the certificate for one of the certificate authorities (CAs) in the signing hierarchy, and that CA is not trusted by the local application.               |
-| `UNABLE_TO_GET_ISSUER_CERT_LOCALLY`  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete.                            |
+| `UNABLE_TO_GET_ISSUER_CERT`          | Unable to find the certificate for one of the certificate authorities (CAs) in the signing hierarchy, and that CA is not trusted by the local application. For example, this error may be thrown when the self-signed Root CA, but not the Intermediate CA, is missing from the list of trusted certificates.               |
+| `UNABLE_TO_GET_ISSUER_CERT_LOCALLY`  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete. For example, this error may be thrown when the self-signed Root CA and Intermediate CA are both missing from the list of trusted certificates.                            |
```
Comment (maycmlee) [inline content/en/synthetics/api_tests/errors.md:52]:
```suggestion
| `UNABLE_TO_GET_ISSUER_CERT`          | Unable to find the certificate for one of the certificate authorities (CAs) in the signing hierarchy, and that CA is not trusted by the local application. For example, this error may be thrown when the self-signed root CA, but not the intermediate CA, is missing from the list of trusted certificates.               |
| `UNABLE_TO_GET_ISSUER_CERT_LOCALLY`  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete. For example, this error may be thrown when the self-signed root CA and intermediate CA are both missing from the list of trusted certificates.                            |
```

**Review by maycmlee — COMMENTED** [review]
A couple of small suggestions

File: content/en/synthetics/api_tests/errors.md (hunk 1 above)
Comment (thang62590) [reply content/en/synthetics/api_tests/errors.md:52]: looks good to me!

Comment (maycmlee) [inline content/en/synthetics/api_tests/errors.md:52]:
```suggestion
| `UNABLE_TO_GET_ISSUER_CERT`          | Unable to find the certificate for one of the certificate authorities (CAs) in the signing hierarchy, and that CA is not trusted by the local application. For example, this error may be thrown when the self-signed root CA, but not the intermediate CA, is missing from the list of trusted certificates.               |
| `UNABLE_TO_GET_ISSUER_CERT_LOCALLY`  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete. For example, this error may be thrown when the self-signed root CA and intermediate CA are both missing from the list of trusted certificates.                            |
```

**Review by maycmlee — APPROVED** [review]

**Review by maycmlee — APPROVED** [review]

**Pull request merged**